packages/shared/components/ProposalQuestion.svelte

```diff
@@ -33,14 +37,15 @@
             <Icon icon={IconEnum.ChevronDown} classes="text-gray-500" />
         </div>
     </div>
-    <proposal-answers class:mt-4={isOpened} class="space-y-2">
-        {#each question.answers as answer, answerIndex}
+    <proposal-answers class:mt-4={isOpened || voteValue} class="space-y-2">
```

Comment:
I think `hasVoteValue` might be better because if the user has abstained then `0` would evaluate to false.

Reply:
The problem is that there is no difference between a user abstaining the vote or never having voted for it. Both would result in `0`